index.js

```diff
@@ -85,8 +86,6 @@ navLink.forEach((link, index) => {
 });
 
 const Time = document.querySelector('.time');
-const date = new Date();
-const format = {
-  month: 'long', weekday: 'long', day: 'numeric', year: 'numeric', hour: '2-digit', minute: '2-digit', second: '2-digit',
-};
-Time.textContent = date.toLocaleTimeString('en-us', format);
+Time.textContent = ` ${DateTime.now().toJSDate().toLocaleTimeString('en-US', {
```

Comment:
- [ ] [OPTIONAL] Kindly consider adding a time interval for this time string also it can be updated every second and display the right time on the page instantly